pkg/addon-operator/operator.go

```diff
@@ -381,7 +402,7 @@ func (op *AddonOperator) RegisterManagerEventsHandlers() {
 			"event.id": uuid.Must(uuid.NewV4()).String(),
 			"binding":  string(htypes.Schedule),
 		}
-		logEntry := log.WithFields(utils.LabelsToLogFields(logLabels))
+		logEntry := utils.EnrichLoggerWithLabels(op.Logger, logLabels)
```

Comment:
I wonder why we can't put the enrichment with lables in the log package? Why do we need to create separate functions in the common utils package?

Reply:
we can add this to the next iteration of logger development.
greate discussion point